gen/go/eventschema/getters.go

```diff
@@ -71,87 +71,135 @@ func QueryableEventList() (string, error) {
 	return sb.String(), nil
 }
 
-// TableSchema returns a CSV description of the event table schema.
-// This explains to the query writer which field are available.
-// This must not be confused with ViewSchema which returns the Athena SQL
-// statements used to build a view extracting content from raw event_data.
-func (event *Event) TableSchema() (string, error) {
+// TableSchemaDetails describe Athena view schema.
+type TableSchemaDetails struct {
+	// Name is view name
+	Name string
+	// SQLViewName is SQL compatible table name.
+	SQLViewName string
+	// Description is Athena view description.
+	Description string
+	// Columns contains information about columns schema.
+	Columns []*ColumnSchemaDetails
+}
+
+// CreateView returns a SQL statement to create an Athena view.
+// This view is used during a user query execution to extract data from the
+// raw event_data column and make data representation more user-friendly.
+func (d *TableSchemaDetails) CreateView() string {
 	sb := strings.Builder{}
-	sb.WriteString("column_name, column_type, description\n")
-	sb.WriteString("event_date, date, is the event date\n")
-	sb.WriteString("event_time, timestamp, is the event time\n")
-	for _, prop := range event.Fields {
-		line, err := prop.TableSchema([]string{prop.Name})
-		if err != nil {
-			return "", trace.Wrap(err)
-		}
-		sb.WriteString(line)
+	sb.WriteString("SELECT\n")
```

Comment:
Roman asked for a comment on the original PR explaining why it is safe to build SQL queries in this manner, but it looks like that was never done. 

I'd like to see it too. This type of code is prone to introducing SQL injection vulnerabilities.

Reply:
I have added comment but in the e repo where CreateView function is used and forgot to update this one. 

Namely access monitoring user can run any Athena SQL Query so protection agains SQL injection doesn't make sense because we are giving a user to run a customer queries
However Access Monitoring is using a separate IAM role that allows only for read only access to Athena events table: https://github.com/gravitational/cloud/pull/6158/files#diff-0a61b4dbc943463cdee2c13f1dec76ce507b262f2a49473d96470c3d8dfd3ea4R309